usaspending_api/broker/management/commands/load_fpds_transactions.py

```diff
@@ -39,60 +41,68 @@ def get_cursor_for_date_query(connection, date, count=False):
             db_cursor.execute(db_query)
         return db_cursor
 
-    def load_fpds_incrementally(self, date):
-        """
-        Handles "increment" loads, which includes deleting transactions so long as a date is provided
-        :param date: if provided, loader will load and delete all transactions starting from this date
-        :return: nothing
-        """
+    def load_fpds_incrementally(self, date: Optional[datetime], chunk_size: int = CHUNK_SIZE) -> None:
+        """Process incremental loads based on a date range or full data loads"""
+
         if date is None:
-            logger.info("fetching all fpds transactions...")
+            logger.info("Skipping deletes. Fetching all fpds transactions...")
         else:
-            logger.info("fetching fpds transactions since {}...".format(str(date)))
+            logger.info(f"Handling fpds transactions since {date}...")
```

Comment:
Is it called handling or fetching?

Reply:
Helps differentiate between processing deletes or not.

Handling is both deletes and upserts